wpilibj/src/main/java/edu/wpi/first/wpilibj/Notifier.java

```diff
@@ -99,7 +99,7 @@ public Notifier(Runnable run) {
                   break;
                 }
 
-                Runnable handler = null;
+                Runnable handler;
```

Comment:
There was a warning about the null assignment being redundant since the variable is assigned to later (also, Java values are null/0 by default anyway).